pydis_site/apps/api/migrations/0092_remove_redirect_filter_list.py

```diff
@@ -0,0 +1,16 @@
+from django.db import migrations
+from django.apps.registry import Apps
+from django.db.backends.base.schema import BaseDatabaseSchemaEditor
+
+
+def forward(apps: Apps, _: BaseDatabaseSchemaEditor) -> None:
+    apps.get_model("api", "FilterList").objects.filter(name="redirect").delete()
```

Comment:
`api.Filter` has `on_delete` set to cascade set on the `filter_list_id` field, so deleting the `FilterList` will delete the Filters for that list automatically.